test/00-feed.js

```diff
@@ -7,12 +7,20 @@ tape('00 feed type', function (t) {
   const values = [
     '@6CAxOI3f+LUOVrbAl0IemqiS7ATpQvr9Mdw9LC4+Uv0=.ed25519', // classic
     'ssb:feed/bendybutt-v1/6CAxOI3f-LUOVrbAl0IemqiS7ATpQvr9Mdw9LC4-Uv0=', // bendy-butt
+    'ssb:feed/gabby-grove/FY5OG311W4j_KPh8H9B2MZt4WSziy_p-ABkKERJdujQ=', // gabby-grove
+    '@6CAxOI3f+LUOVrbAl0IemqiS7ATpQvr9Mdw9LC4+Uv0=.ggfeed-v1', // gabby-grove (sigil)
```

Comment:
This should be an unhappy path I suppose

Reply:
Right, I realised that after posting (Friday brainz). There is already a good test there (unhappy path). Now we just need to add a check of the error message from the throw.